ci/cron/src/Main.hs

```diff
@@ -344,8 +344,6 @@ verify_signatures :: FilePath -> FilePath -> String -> IO ()
 verify_signatures bash_lib tmp version_tag = do
     System.callCommand $ unlines ["bash -c '",
         "set -euo pipefail",
-        "export DADE_SKIP_JAVA=1",
```

Comment:
should we remove the skip java logic again given that it’s now unused?

Reply:
Good point.